quickfixj-core/src/main/java/quickfix/Session.java

```diff
@@ -423,6 +428,7 @@
     private boolean enableNextExpectedMsgSeqNum = false;
     private boolean enableLastMsgSeqNumProcessed = false;
     private boolean validateChecksum = true;
+    private boolean allowPosDup;
```

Comment:
```suggestion
    private boolean allowPosDup = false;
```
I know `false` is the default but let's make it explicit as it is with the other flags.